### DISCUSSION TITLE
Search Box Cache Configuration ?

### DISCUSSION THREAD

**Discussion by piranhap** [microsoft-search/pnp-modern-search#3056] [Q&A] [answered]
Hello! 
I have a search box with a search results page setup as well. If someone looks for an item let's say VPN it takes him there but if after they click on go back on their browser the search page is clean. 
Is there any way to have the search box or search results remember the text query unless they click on clear? 
I hope that makes sense I suck at explaining things. 
Thanks!

**Comment by wobba** [comment, marked as answer]
See https://learn.microsoft.com/en-us/sharepoint/dev/spfx/hyperlinking on how to make hyperlinks work better. And opening the link in a new tab might be a better experience as well.
